include/mata/nfa-algorithms.hh

```diff
@@ -44,7 +44,7 @@ namespace Algorithms {
     Nfa complement_classical(
             const Nfa&         aut,
             const Alphabet&    alphabet,
-            SubsetMap*         subset_map);
+            std::unordered_map<StateSet, State>* subset_map);
```

Comment:
I'm wondering, whether it would be helpful to include `using std;` at the begging of the file to shorten the type to `unordered_map<T1, T2>`. @ondrik @Adda0 @martinhruska what do you think?

I don't think anyone will clash this type with `Mata::unordered_map`.

Reply:
As already said, definitely not in the header file. I do not like using `uning namespace` directive in general, but I can work with that in the source files, if you think it necessary. I do not like that, however, especially in a case of standard library. If it is our own data types we include with `using namespace`, that is fine by me, but not standard library. Nevertheless, as I have already said, I can work with that.

What about `using std::unordered_map;`, instead, if you must use anything? See [SO this suggestion in this SO topic](https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice#comment1302620_1452721).

Reply:
I don't like `using namespace` too, thefore I don't use it. Does anyone strongly want to use it?

Reply:
Consider this resolved then. I don't like the using either, but I was thinking that at some parts the signatures were quite complicated and could scare Mr. L.